pallets/rewards/src/tests/currency_movement.rs

```diff
@@ -47,6 +56,15 @@ macro_rules! currency_movement_tests {
 				}
 			}
 
+			fn has_currencies(user: u64, currency_ids: &[(DomainId, CurrencyId)]) {
```

Comment:
I think rust allows to check the equality of two slices directly. Something like:
```rust
assert_eq!(&$pallet::list_currencies(user), currency_ids);
```

Reply:
Dropped that ugly func and used that assert instead.